src/main/java/org/fisco/bcos/channel/handler/ConnectionInfo.java

```diff
@@ -44,10 +48,16 @@ public void setTopics(List<String> topics) {
         this.topics = topics;
     }
 
-    private String nodeID = "";
+    @Deprecated private String nodeID = "";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4981260)